index.html

```diff
@@ -74,29 +74,24 @@
             <div class="row">
 
                 <!-- FOOTER MENU -->
-                <div class="col-md-6">
+                <div class="col-md-12">
                     <div class="footer-menu">
-                        <ul class="list-inline">
-                            <li><a href="#">privacy policy</a>
-                            </li>
-                            <li><a href="#">terms of service</a>
-                            </li>
-                            <li><a href="#">support</a>
-                            </li>
-                            <li><img src="img/Twitter.png"</a>
-                            </li>
-                        </ul>
+                        <div class="col-md-3">
+                            <a href="#">privacy policy</a>
+                        </div>
+                        <div class="col-md-3">
+                            <a href="#">terms of service</a>
+                        </div>
+                        <div class="col-md-3">
+                            <a href="#">support</a>
+                        </div>
+                        <div class="col-md-3">
+                            <img src="img/Twitter.png"</a>
+                        </div>
```

Comment:
I found this [link](http://revillweb.github.io/bootstrap-tutorial/) and used it to help me (good ole inspect element).

Since we're using twitter bootstrap CSS library, it was an easy fix because the css rules are already there in `col-md-12` and `col-md-3`. You should check it out @joangonz !